core-localstorage.html

```diff
@@ -103,12 +146,13 @@
           try {
             v = JSON.parse(v);
           } catch(x) {
+            console.error("could not parse local storage value", v);
           }
           this.value = v;
         }
       }
       this.loaded = true;
-      this.asyncFire('core-localstorage-load');
+      this.fire('core-localstorage-load');
```

Comment:
Is this always implied every time that `value` changes? What if it is set externally?